### ISSUE TITLE
New task with label is missing from list of tasks with that label

### ISSUE THREAD

**Issue by rotalumi** [alainm23/planify#861]
**Description of the bug**

A new task, to which a label has been assigned, does not immediately appear in the list of tasks with that label.

**How to reproduce**

1. Create a new task in the Inbox and assign to it an existing label.
2. Notice that in the sidebar, the Inbox count increased by 1 and the label's count increased by 1. Both counts are correct.
3. In the sidebar, click the label we had assigned to our new task.
4. Notice that the list of tasks (having this label) is missing the task we just created.

**Note:** Any further tasks added to the Inbox with the same label also do not appear in the list, although the count increments each time as expected.

**Note:** Clicking other objects in the sidebar at this point does not correct the problem. However, see the workaround below, which necessitates restarting Planner.

**How to work around the problem**

1. With the focus still on that label in the sidebar, close and then reopen Planner.
2. When Planner reopens, notice that the focus is still on that label, and the list of tasks is still missing the task we just created.
3. However, after clicking other objects in the sidebar and then returning to the label, notice that our new task suddenly appears in the list of tasks with that label.

**Note:** I haven't found any way to work around the problem except to close and reopen Planner.

**Expected behavior**

When a new task is created with a label, it should immediately appear in the list of tasks with that label, without the need to restart Planner.

**My desktop environment**

I am using a Flatpak version of Planner on Arch Linux. I use the KDE desktop environment.

Although the current Flatpak version presents itself as version 2.7, there have been several updates since the original version 2.7. I believe I am actually using version 3.0.5. I'd have no way to know which version I am using except that one of those updates resulted in the loss of all of my data. Planner's Twitter feed informed me, after I lost all of my data, that I might lose all of my data with an upgrade to a version prior to 3.0.5:

> If you are using Planner in Flathub, we are having data loss problem when upgrading to v3.0.3, please do not start Planner until upgrading to v3.0.5.

Therefore, I must have upgraded to one of the problematic updates, even though the update manifest says that I was upgrading to version 2.7.

**Note:** I am not a Twitter user, so I wasn't warned prior to the upgrade that I was about to lose all of my Planner data.

**Note**: Unlike almost every other app I use, I don't see in Planner the version I'm using, like in the **Help > About** menu item.

I have installed the latest Flatpak update, which I think is version 3.0.5, but I don't really know. The Flatpak update has told me with every recent update that I'm updating to version 2.7, which is clearly wrong.
